app/views/shared/_menu-tab.html.erb

```diff
@@ -1,16 +1,20 @@
 <div class="container">
   <div class="menu-tabs fixed-bottom">
     <div class="pantry-tab">
-      <h4><%= link_to ('<i class="far fa-plus-square"></i></i>').html_safe, new_item_amount_path, class: "pantry-tab" %></h4>
+      <h4><%= link_to ('<i class="far fa-plus-square"></i>').html_safe, new_item_amount_path, class: "pantry-tab" %></h4>
+      <p>New item</p>
```

Comment:
For symmetry, can we change this to "Add item"?

Reply:
@worksofgenius okay it's updated. and also the conflicts have been resolved.